samples/built-in-policy/audit-classic-VM/azurepolicy.json

```diff
@@ -0,0 +1,33 @@
+{
+    "properties": {
+        "displayName": "Audit use of classic virtual machines",
+        "policyType": "BuiltIn",
+        "mode": "All",
+        "description": "Use new Azure Resource Manager v2 for your virtual machines to provide security enhancements such as: stronger access control (RBAC), better auditing, ARM-based deployment and governance, access to managed identities, access to key vault for secrets, Azure AD-based authentication and support for tags and resource groups for easier security management",
+        "metadata": {
+            "category": "Compute"
```

Comment:
I know this is in conflict with some previous guidance (sorry!), but we've decided that all the samples should be organized by category, regardless of whether they were originally a built-in or not.

So instead of being in the `samples/built-in-policy` folder, this one should be in `samples/Compute`,etc.

If you have any trouble figuring out what category the policies should be in ,let me know and we can figure it out offline.